dlc-manager/tests/manager_execution_tests.rs

```diff
@@ -660,20 +787,70 @@ fn manager_execution_test(test_params: TestParams, path: TestPath) {
 
             match path {
                 TestPath::Close => {
-                    periodic_check!(first, contract_id, PreClosed);
+                    let case = thread_rng().next_u64() % 3;
+                    let blocks: Option<u32> = if case == 2 {
+                        Some(6)
+                    } else if case == 1 {
+                        Some(1)
+                    } else {
+                        None
+                    };
+
+                    if manual_close {
+                        periodic_check!(first, contract_id, Confirmed);
+
+                        let attestations = test_params
+                            .oracles
+                            .iter()
+                            .enumerate()
+                            .flat_map(|(i, o)| o.get_attestation(EVENT_ID).ok().map(|a| (i, a)))
+                            .take(
+                                test_params
+                                    .contract_input
+                                    .contract_infos
+                                    .first()
+                                    .unwrap()
+                                    .oracles
+                                    .threshold as usize,
+                            )
```

Comment:
needed to add this take for the multi contract info DLCs, not sure if there is a cleaner way

Reply:
got a cleaner way now